CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
-2.22.2 (In Progress)
+2.22.2 (July 25, 2022)
 ====================
```

Comment:
Super small nit: To extend the `=` to match the length of the date. 

Otherwise, LGTM! +1